### ISSUE TITLE
Suggestion: Standardize Id and names for types

### ISSUE THREAD

**Issue by DatTronGuy** [digin-energi/Grunnprofil#302]
Might be outside of the scope for the project, but I think it would be a good idea to also standardize common elements where possible.

Working with data from multiple companies, we see alot of the same data being represented by different names and ID's:

Example - ConductorTypes:
Customer 1:
Id: 1, Name: "1x3xEX50"
Id: 2, Name: "1x2xCU16"

Customer 2:
Id: 334, Name: "1_3xEx50"
Id: 225, Name: "1_2xCU16"

We also see cases where the same data has different names and ID's within the same company:

Company 2 - continued:
Id: 226, Name: "1 2xCU16"
Id: 227, Name: "1___2xCU16"

Etc, etc.

**Comment by Sveino** [comment]
This is inside the scope of DIGIN, but a bit more related to [Produktdata](https://github.com/digin-energi/produktdata-public). This will be address under topic of Reference data.

**Comment by DatTronGuy** [comment]
Okay.

Do you want me to move the issue to the ProduktData repository?

**Comment by Sveino** [comment]
No. It is OK to have it here. The principle will be documented in this repo, but the concrete reference data will follow the ProduktData repository.
